test/plugin/containers/jvm-container/pom.xml

```diff
@@ -52,7 +52,8 @@
                                 <runCmds>
                                     <run>chmod +x /usr/local/skywalking/run.sh</run>
                                     <run>tar -xvf ../tools/skywalking-mock-collector.tar.gz -C ../tools</run>
-                                    <run>apk add --no-cache bash curl</run>
+                                    <run>apt-get update -y</run>
+                                    <run>apt-get install -y unzip</run>
```

Comment:
Hi, where is `unzip` used?

Reply:
test/plugin/containers/jvm-container/src/main/docker/run.sh
line 64
```
unzip -q ${SCENARIO_HOME}/*.zip -d /var/run/
if [[ ! -f /var/run/${SCENARIO_NAME}/${SCENARIO_START_SCRIPT} ]]; then
    exitOnError "The required startup script not exists!"
fi
```